docs/theory/heat_physics.rst

```diff
@@ -0,0 +1,372 @@
+.. _chp_heat_physics:
+
+Heat Physics
+============
+
+MESIDO models thermal power (:math:`\dot{Q}`), volumetric flow (:math:`\dot{V}`), and head (:math:`H`) for DHS systems.
+These variables are modelled over all timesteps :math:`K` and for each asset in the system :math:`A`.
+An asset :math:`a \in A` in the system is modelled using a set of in-ports, denoted by :math:`I^a_{in} \neq \emptyset`, and a set of out-ports, denoted by :math:`I^a_{out} \neq \emptyset`, with :math:`I^a` denoting the union of these two sets.
+For an asset :math:`a` the three variables :math:`\dot{Q}_i, \dot{V}_i`, and :math:`H_i` denote the thermal power, volumetric flow and head for port :math:`i \in I^a`. For readability, the variables and equations of the model, given below, are defined without indexing the variables and sets for each timestep.
+The equations below are assumed to hold for every timestep, unless otherwise specified.
+
+Modelling specifically these three variables allows to ensure a hydraulically feasible solution whilst linearizing the physics.
+Energy changes are modelled with the assumption of a constant temperature throughout the network.
+Although this assumption is not realistic, the linearization made will show that the effects of these assumptions are relatively small and conservative to the objective function outcome.
+
+General Physics
+---------------
+
+The DHS is modelled as a closed loop system and thus at every timestep and for every asset, mass balance should hold.
+The system is modeled with constant temperature and thus constant density and specific heat.
+The mass balance is provided as a volumetric flow balance in :eq:`eq:flow_balance`.
+
+.. math::
+    :label: eq:flow_balance
+
+    \sum_{i \in I^a} \dot{V}_{i} = 0 \;\; \forall a \in A
+
+
+To garuantee energy balance :eq:`eq:general_energy_balance` is applied.
+
+.. math::
+    :label: eq:general_energy_balance
+
+    \sum_{i \in I} \dot{Q}_i + \dot{Q}^a_{consumed} = 0 \;\; \forall a \in A
+
+:math:`\dot{Q}^a_{consumed}` is the thermal power consumed by asset :math:`a`.
+The power losses for the asset are included in this variable. Furthermore, the convention that positive consumed power is equal to thermal power production of an asset.
```

Comment:
This is confusing as below at the source it is stated that Qaconsumed is equal to the (negative) value of the produced heat.

Reply:
The last sentence has been removed